src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -30,177 +30,177 @@
+            TEST_WIKI_SKTEST123,
             "Special:Maps",
             "Maps - Sktest123 Wiki - Wikia",
         },
         {
-            TEST_WIKI_ORIGINAL_TITLE,
+            TEST_WIKI_SKTEST123,
             "Template:Welcome",
             "Template:Welcome - Sktest123 Wiki - Wikia",
         },
         {
-            TEST_WIKI_ORIGINAL_TITLE,
+            TEST_WIKI_SKTEST123,
             "MediaWiki:Common.css",
             "MediaWiki:Common.css - Sktest123 Wiki - Wikia",
         },
         {
-            TEST_WIKI_ORIGINAL_TITLE,
+            TEST_WIKI_SKTEST123,
             "MediaWiki:Edit",
             "MediaWiki:Edit - Sktest123 Wiki - Wikia",
         },
         {
-            TEST_WIKI_ORIGINAL_TITLE,
+            TEST_WIKI_SKTEST123,
             "User:Sktest",
             "User:Sktest - Sktest123 Wiki - Wikia",
         },
         {
-            TEST_WIKI_ORIGINAL_TITLE,
+            TEST_WIKI_SKTEST123,
             "File:THE_HOBBIT_Trailer_-_2012_Movie_-_Official_HD",
             "Video - THE HOBBIT Trailer - 2012 Movie - Official HD - Sktest123 Wiki - Wikia",
         },
         {
-            TEST_WIKI_ORIGINAL_TITLE,
+            TEST_WIKI_SKTEST123,
             "File:Giant_prominence_on_the_sun_erupted.jpg",
             "Image - Giant prominence on the sun erupted.jpg - Sktest123 Wiki - Wikia",
         },
         {
-            TEST_WIKI_ORIGINAL_TITLE,
+            TEST_WIKI_SKTEST123,
             "Special:NonExistingSpecialPage",
             "Error - Sktest123 Wiki - Wikia",
         },
         // Em-dash instead of regular dash for some languages
         {
-            TEST_WIKI_ORIGINAL_TITLE_WITH_EM_DASH,
+            TEST_WIKI_POZNAN,
             "Ulica_Kazimierza_Wielkiego",
-            "Ulica Kazimierza Wielkiego – Poznańska Wiki – Wikia",
+            "Ulica Kazimierza Wielkiego - Poznańska Wiki - Wikia",
         },
         // Custom title
         {
-            TEST_WIKI_CUSTOM_TITLE,
+            TEST_WIKI_ES_POKEMON,
             "WikiDex",
             "WikiDex, la enciclopedia Pokémon - Wikia",
         },
         {
-            TEST_WIKI_CUSTOM_TITLE,
+            TEST_WIKI_ES_POKEMON,
             "Lista_de_Pokémon",
             "Lista de Pokémon - WikiDex, la enciclopedia Pokémon - Wikia",
         },
         {
-            TEST_WIKI_CUSTOM_TITLE,
+            TEST_WIKI_ES_POKEMON,
             "Special:UnusedVideos",
             "Unused videos - WikiDex, la enciclopedia Pokémon - Wikia",
         },
         {
-            TEST_WIKI_CUSTOM_TITLE,
+            TEST_WIKI_ES_POKEMON,
             "Categoría:Regiones",
             "Categoría:Regiones - WikiDex, la enciclopedia Pokémon - Wikia",
         },
         // Corporate pages
         {
-            TEST_WIKI_CORPORATE,
+            TEST_WIKI_WWW,
             "",
-            "Collaborative communities for everyone! - Wikia",
+            "Wikia",
         },
         /* MAIN-5823 {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)